src/docs/src/whatsnew/3.3.rst

```diff
@@ -0,0 +1,263 @@
+
+* :ghissue:`3832`: Add ``GET`` variant to ``_dbs_info`` endpoint, used to be ``POST`` only.
+
+* :ghissue:`3864`: Improve ``erlang_ls`` configuration.
+
+* :ghissue:`3853`: Remove legacy ``ddoc_cache_opener`` gen_server and speed up event routing.
+
+* :ghissue:`3879`: Remove use of ``ERL_OPTS`` environment variable. All supported Erlang version now use ``ERL_COMPILER_OPTIONS`` for the same purpose.
+
+* :ghissue:`3883`: Add support for SpiderMonkey 91.
+
+* :ghissue:`3889`: Track ``libicu`` collator versions in the view header
+
+* :ghissue:`3925`: Make the timeout for receiving requests from attachment writers configurable.
+
+* :ghissue:`3927`: Include index signature in ``_search_info``.
+
+* :ghissue:`3963`: Optimtize key tree stemming by using maps instead of sets. This greatly reduced memory usage for heavily conflicted docs in some situations.
+
+* :ghissue:`3974`: Create new config options in ``[couchdb]`` and ``[smoosh]`` sections to enable finer control of compaction logging levels.
+
+* :ghissue:`3983`, :ghissue:`3984`, :ghissue:`3985`, :ghissue:`3987` :ghissue:`#4033`: Add various functions to ``couch_debug`` module.
+
+* :ghissue:`4000`: Ensure ``Object.prototype.toSource()`` is always available
+
+* :ghissue:`4017`: Update ``mochiweb`` to 3.1.1.
+
+* :ghissue:`4018`: Update ``jiffy`` to 1.1.1 and ``b64url`` to 1.0.3.
+
+* :ghissue:`4021`: Reduce smoosh compaction log level to ``debug``.
+
+* :ghissue:`4135`: Allow and evaluate nested json claim roles in JWT token.
+
+* :ghissue:`4060`: Add support for Erlang 25.
+
+* :ghissue:`4064`: Enable replicating purge requests between nodes. Also avoid apploying interactive purges more than once.
+
+* :ghissue:`4069`, :ghissue:`4084`: Drop support for Erlang < 23, update ``vm.args`` settings to match. Review this if you have customized your ``vm.args``.
+
+* :ghissue:`4083`: Support Elixir 13.
+
+* :ghissue:`4085`: Add an option to let ``custodian`` always use ``[cluster] n`` value.
+
+* :ghissue:`4095`: Implement `winning_revs_only` option for the replicator. It replicates only the winning revisions from the source to the target, effectively discarding conflicts.
+
+* :ghissue:`4135`: Separate search IO from file IO.
+
+* :ghissue:`4140`, :ghissue:`4162`: Upgrade hash algorithm for cookie auth (sha1 -> sha256). This introduces a new config setting ``hash_algorithms``. New cookie values are hashed with sha256, sha1 hashes are still accepted. Admins can set this to sha256 only. Sha1 will be disallowed in the next major release. Show supported hash algorithms in ``_versions`` endpoint.
+
+* :ghissue:`4179`: Don't double-encode changes sequence strings in the replicator.
+
+* :ghissue:`4182`: Explicitly maintain a fully connected cluster. Previously, it was possible for the nodes to disconnect, and for that state to persist util the nodes restarted.
+
+* :ghissue:`4177`: Redact passwords in log file.
```

Comment:
#4177 is: Remove the long deprecated bigcouch 0.4 change sequence support 

#4198 Redact passwords in log file